### DISCUSSION TITLE
2.8.3 is available in public?

### DISCUSSION THREAD

**Discussion by tmk** [NicoHood/HID#376] [General]
Hi all,
Thank you contributers for your work in this great project.

It seems to be version 2.8.3 is ready in this repository, but it is not available yet for library manager in Arduino IDE and arduino-cli. Is there any reason for this?

**Comment by NicoHood** [comment]
I guess because I forgot to bump this version number: https://github.com/NicoHood/HID/blob/master/library.properties

**Reply by NicoHood** [reply]
I fixed it now

**Reply by tmk** [reply]
Thank you! I upgraded to 2.8.4 successfully.
